breaking-changes-guide.md

```diff
@@ -91,8 +91,10 @@ Before flipping the default value of the flag to true, please make sure that:
 
 When changing the flag default to true, please:
 
-  * Use [`RELNOTES:`](release-notes.html) in the commit description, with the
-    name the of the flag.
+  * Use [`RELNOTES[INC]:`](release-notes.html) in the commit description, with the
```

Comment:
Most people don't use `RELNOTES[INC]` and I don't think it's actually useful. I'd rather remove it and use only `RELNOTES`.

(adding a tag like `C++`, `Starlark`... would be more useful than `INC` and would simplify our work)

Reply:
+1 I'm an advocate for topical RELNOTES.

Reply:
I agree that people don't use [INC], my hope is that changing this document will _cause_ people to use it :) 

By using [INC] in the "I just flipped an incompatible change flag" case, it groups all such issues in the "Incompatible Changes" section of the auto-generated release notes, which is where they're supposed to be.

Reply:
+1 If we have tools that do something useful with [INC]  then we should certainly encourage people to make use of it. 
My only nit would be that the : belongs outside the link rather than in it. But, why is this even a link to release-notes.html? Following the link won't clarify why you are adding the RELNOTES. Except obliquely, when you infer that adding RELNOTES puts it into release_notes.html.

Reply:
@aiuto I updated the link

Reply:
cc @fweikert who worked / wrote proposals about relnotes.

My point is that we used to encourage using INC and it didn't work (too much confusion about the different relnotes tags and typing just `RELNOTES:` is easier). At the same time, Florian has a proposal to change the relnotes tags, although I'm not sure what's the current state.

So my point is: if we want to change the habits, we should first decide where we want to go (and make sure the change is aligned).

Reply:
@fweikert, any thoughts? Otherwise I'm going to submit the PR tomorrow